modules/runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/model/impl/ComponentManagerImpl.java

```diff
@@ -509,14 +510,26 @@ protected static void handleError(String message, Exception e) {
         log.info("Activate components");
         Watch watch = new Watch();
         watch.start();
+
+        // first instantiate resolved components: that allows some to register as listeners on ComponentManager before
+        // all components activation.
+        List<RegistrationInfo> iris = new ArrayList<>();
+        // first activate resolved components
+        for (RegistrationInfo ri : registry.getResolvedRegistrationInfo()) {
+            watch.start(ri.getName().getName());
```

Comment:
The watch calls on this step will be erased by the _activate_ step.
Here we should declare a new Watch, if we're interested, and write its metrics to another file or not do it.

Reply:
Indeed I thought they would addition (since component instantiation could be costly), so i'll add a new watch.